sympy/sets/sets.py

```diff
@@ -278,10 +278,10 @@ def contains(self, other):
 
         """
         other = sympify(other, strict=True)
-        ret = self._contains(other)
+        ret = sympify(self._contains(other))
```

Comment:
I am not sure about that. But i must mention that i get a different traceback each time. 
```
>>> from sympy import *
>>> (S.Complexes).union(S.Reals)
^CTraceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 85, in union
    return Union(self, other)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 1181, in __new__
    return Union.reduce(args)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 1215, in reduce
    new_set = s._union(t)
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 953, in _union
    if ((self.left_open and other.contains(self.start) is true) or
  File "/home/gxyd/Public/sympy/sympy/sets/sets.py", line 281, in contains
    ret = sympify(self._contains(other))
  File "/home/gxyd/Public/sympy/sympy/sets/fancysets.py", line 827, in _contains
    from sympy.functions import arg, Abs
  File "<frozen importlib._bootstrap>", line 2264, in _handle_fromlist
KeyboardInterrupt
```